.gitignore

```diff
@@ -41,3 +41,4 @@ TAGS
 # envdir/envrc (https://direnv.net)
 /.envrc
 /bin
+/compose-data/*
```

Comment:
This could be named just "data" in the new git repo.

I noticed one thing, when you use podman as non-root, postgres will change permissions of the directory to container user (random uid) and 600 permissions which makes the directory undeletable for the hosting user. It does not seem to be a problem for podman itself. There is a nice trick to fix this:

    podman unshare chmod 777 data/*

Let’s just document this.